src/main/java/duke/Storage.java

```diff
@@ -0,0 +1,115 @@
+package duke;
+
+import java.io.File;
+import java.io.FileWriter;
+import java.io.IOException;
+import java.nio.file.Files;
+import java.nio.file.Paths;
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Objects;
+
+/**
+ * Used by the TaskList class, and handles reading, writing and deleting from lines in the file
+ */
+public class Storage {
+
+    private String filepath;
+
+    /**
+     * Carries out operations like reading and writing to a specified database file
```

Comment:
Should there be a line break before the tags? I noticed the same issue in other places as well.